modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -49,6 +50,11 @@ The following values for `operatorLogLevel` are supported:
 |Defines the Proxy to be used when calling master API
 and upstream registries.
 
+|`affinity`
+|You can use the `affinity` configuration parameter to configure pod scheduling preferences and constraints for Image Registry Operator pods.
```

Comment:
Most of the other descriptions in the table start with a stronger word, often a verb. Maybe make this one more consistent. (Plus, it removes two of the three configure/configuration uses in the table cell.)
```suggestion
|Configures pod scheduling preferences and constraints for Image Registry Operator pods.
```

Reply:
I would argue that "you can use" is more active than "configures". I think starting the lines with verbs like this also blurs the lines a bit wrt anthropomorphism, since the setting itself doesn't configure this, you use the setting to configure this 😉 I could reduce "configuration parameter" to just "parameter" though, so I'll update that 😄